terraform/shared/modules/env/postgrest.tf

```diff
@@ -22,7 +22,7 @@ resource "cloudfoundry_app" "postgrest" {
   space        = data.cloudfoundry_space.apps.id
   docker_image = "ghcr.io/gsa-tts/fac/postgrest@${data.docker_registry_image.postgrest.sha256_digest}"
   timeout      = 180
-  memory       = 512
+  memory       = 1024
```

Comment:
I'm not certain whether this should still be in MB.

Reply:
I'm not ready to conduct an experiment otherwise. But, yes, the numbers get big.